packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -445,7 +466,13 @@ const makeVaultDirector = defineDurableFarClassKit(
           burnDebt,
         });
 
-        const vm = makeVaultManager(
+        const { managerBaggages } = ephemera;
+        const makeVaultManager = managerBaggages.addChild(
+          collateralBrand,
```

Comment:
this should be the brand name, rather than the brand itself.

Reply:
what's wrong with using the Brand object?

if using the name, it would be only alleged name. why is that safe?

Reply:
from discussion: it's just a diagnostic string and it's being stringified anyway. better to use use the alleged name because it will be cleaner in output.